session-replacement/src/main/java/com/amadeus/session/repository/redis/RedisConfiguration.java

```diff
@@ -113,6 +139,12 @@
 
   String port;
 
+  String password;
+
+  boolean useSSL = false;
```

Comment:
How about using `Boolean` and assigning to `null` as most other attributes are? It could allow  forcing to false through configuration string.

Reply:
Agreed. Will use `Boolean` and assign it to `null`